### PR TITLE
Backport-2.4-1048 AAP-10975B Add Event-Driven content to existing RH AAP Installation guide

### PR DESCRIPTION
**Pull request by jself-sudoku**
Added EDA content in the following areas of the existing Installation Guide:

- Chapter 1 - introductory paragraph; also added cross-reference to verification process for EDA controller (in Chapter 2)
- Chapter 2 - Added EDA scenario (Single Event-Driven Ansible controller node with internal db) to list of hyperlinks
     - Section 2.1.1.4 - Added EDA code example at the end of the code example for this section.
     - Section 2.1.1.5 - Added EDA code example at the end of the code example for this section.
     - Section 2.1.1.6 - Created new EDA code example.
     - Section 2.5 - Verifying EDA installation.
- Appendix A. - Added include:: syntax for EDA.
     - Appendix A.5. - new EDA appendix, "Event-Driven Ansible controller variables".

### CODE REVIEW

File: downstream/modules/platform/ref-eda-controller-variables.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Default = false
+//Add this variable back for the next release, as long as approved by development.
+//| *`automationedacontroller_websocket_ssl_verify`* | 
+//SSL verification for the Daphne websocket used by podman to communicate from the pod to the host. Default is false to disable SSL connection as verified
+
+//Default = false
+| *`automationedacontroller_gunicorn_workers`* | Number of workers for the API served through gunicorn.
+
+Default = (# of cores or threads) * 2 + 1
```
Comment (ianf77) [inline downstream/modules/platform/ref-eda-controller-variables.adoc:39]: This looks like ( ...) x3, would 2*( ...) +1 be clearer?

File: downstream/modules/platform/ref-example-platform-ext-database-customer-provided.adoc [hunk 2]
```diff
@@ -96,4 +96,52 @@ automationhub_pg_sslmode='prefer'
...
+# automation_controller_main_url=''
+
+# Boolean flag used to verify Automation Controller's
+# web certificates when making calls from Automation EDA Controller.
+#
+# automationedacontroller_controller_verify_ssl = true
+
+# SSL-related variables
+
+# If set, this will install a custom CA certificate to the system trust store.
```
Comment (ianf77) [inline downstream/modules/platform/ref-example-platform-ext-database-customer-provided.adoc:125]: this installs

File: downstream/modules/platform/ref-example-platform-ext-database-inventory.adoc [hunk 3]
```diff
@@ -94,4 +94,53 @@ automationhub_pg_sslmode='prefer'
...
+
+automationedacontroller_pg_host='data.acme.org'
+automationedacontroller_pg_port=5432
+
+automationedacontroller_pg_database='automationedacontroller'
+automationedacontroller_pg_username='automationedacontroller'
+automationedacontroller_pg_password='<password>'
+
+# The full routable URL used by EDA to connect to a controller host.
+# This URL is required if there is no Automation Controller configured
```
Comment (ianf77) [inline downstream/modules/platform/ref-example-platform-ext-database-inventory.adoc:111]: {ControllerName}

File: downstream/modules/platform/ref-example-platform-ext-database-inventory.adoc [hunk 4]
```diff
@@ -94,4 +94,53 @@ automationhub_pg_sslmode='prefer'
...
+# in inventory.
+#
+# automation_controller_main_url=''
+
+# Boolean flag used to verify Automation Controller's
```
Comment (ianf77) [inline downstream/modules/platform/ref-example-platform-ext-database-inventory.adoc:116]: {ControllerName}

File: downstream/modules/platform/ref-eda-controller-variables.adoc (hunk 1 above)
Comment (jamesmarshall24) [reply downstream/modules/platform/ref-eda-controller-variables.adoc:39]: Order of operations its fine, but I could understand how someone could read it in their head as * 3. If you moved the 2 to the front that'd be fine: `2 * (# of cores of threads) + 1` just as long as 2 is spaced enough that they can see it and not overlook.

Comment (jself-sudoku) [reply downstream/modules/platform/ref-eda-controller-variables.adoc:39]: Hi @ianf77. I think I need to check with James Marshaller (Installation SME) to see if he agrees with that proposal.

Comment (jself-sudoku) [reply downstream/modules/platform/ref-eda-controller-variables.adoc:39]: Okay, he is agreeable to the update. Will update in a bit.

File: downstream/modules/platform/ref-example-platform-ext-database-customer-provided.adoc (hunk 2 above)
Comment (jself-sudoku) [reply downstream/modules/platform/ref-example-platform-ext-database-customer-provided.adoc:125]: Thanks for catching the future tense.

**Review by dcdacosta — APPROVED** [review]
LGTM!

**Pull request merged**